### PR TITLE
feat(sdk): add `login` and `logout` sdk

### PR DESCRIPTION
**Pull request by tianweidut**
## Modules
- [x] Python-SDK

## Checklist
- [x] run code format and lint check
- [x] add unit test
- [ ] add necessary doc

### CODE REVIEW

File: client/starwhale/api/_impl/instance.py [hunk 1]
```diff
@@ -0,0 +1,149 @@
...
+    username: str = "",
+    password: str = "",
+    token: str = "",
+) -> None:
+    """Login to a Starwhale Server instance or Cloud instance.
+
+    Provide either username + password or token to login. Login information will be saved in `~/.config/starwhale/config.yaml`.
+    login function is equivalent to `swcli instance login` command.
+
+    We cannot login to a Standalone instance.
```
Comment (anda-ren) [inline client/starwhale/api/_impl/instance.py:29]: Also add a comment that users could only have one logined instance at a time. That is a concern for multi-instance operating.

**Comment by goldenxinxing** [comment]
Is the concept of session more friendly to multi-user operations? Or optimize later

**Comment by tianweidut** [comment]
> Is the concept of session more friendly to multi-user operations? Or optimize later

no session plan. The current login information is isolated by the Linux Account.  @goldenxinxing

**Review by goldenxinxing — APPROVED** [review]
LGTM

**Pull request merged**